### PR TITLE
[Idle Detection article] Update for launch

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Update for launch.

### CODE REVIEW

File: src/site/content/en/blog/idle-detection/index.md [hunk 1]
```diff
@@ -180,7 +153,7 @@ console.log('IdleDetector is stopped.');
 
 ### DevTools support
 
-Starting in Chrome&nbsp;86, you can emulate idle events in Chrome DevTools without actually being idle.
+Starting in Chromium&nbsp;86, you can emulate idle events in DevTools without actually being idle.
```
Comment (samthor) [inline src/site/content/en/blog/idle-detection/index.md:156]: Should you update to the lauch version, rather than the original flagged versoin?

**Review by samthor — APPROVED** [review]

File: src/site/content/en/blog/idle-detection/index.md (hunk 1 above)
Comment (tomayac) [reply src/site/content/en/blog/idle-detection/index.md:156]: Well, it was available as early as 86, so I think it's fine to leave it as is.

**Comment by tomayac** [comment]
@jpmedley Please merge this on time for the beta blog post.

File: src/site/content/en/blog/idle-detection/index.md [hunk 2]
```diff
@@ -52,33 +45,13 @@ can limit these calculations to moments when the user interacts with their devic
 | 1. Create explainer                      | [Complete][explainer]    |
 | 2. Create initial draft of specification | [Complete][specification]|
 | 3. Gather feedback & iterate on design   | [In progress](#feedback) |
-| 4. Origin trial                          | [In progress][ot]        |
-| 5. Launch                                | Not started              |
+| 4. Origin trial                          | Completed                |
+| 5. **Launch**                            | **Completed**            |
```
Comment (jpmedley) [inline src/site/content/en/blog/idle-detection/index.md:49]: Is this what we always do? Seems like we should say version it landed in.

File: src/site/content/en/blog/idle-detection/index.md (hunk 1 above)
Comment (jpmedley) [reply src/site/content/en/blog/idle-detection/index.md:156]: Because it was an origin trial, it's not usable in anything before 94. Please change.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/idle-detection/index.md (hunk 2 above)
Comment (tomayac) [reply src/site/content/en/blog/idle-detection/index.md:49]: Now we'd use the new widget ([example](https://web.dev/file-system-access/#browser-support)), but the compatibility data for Idle Detection is not yet on MDN.

File: src/site/content/en/blog/idle-detection/index.md (hunk 1 above)
Comment (tomayac) [inline src/site/content/en/blog/idle-detection/index.md:156]:
```suggestion
Starting in Chromium&nbsp;94, you can emulate idle events in DevTools without actually being idle.
```

Comment (tomayac) [reply src/site/content/en/blog/idle-detection/index.md]: Changed then.

File: src/site/content/en/blog/idle-detection/index.md [hunk 3]
```diff
@@ -8,23 +8,16 @@ description: |
   interaction with the keyboard, mouse, screen, activation of a screensaver, locking of the screen,
   or moving to a different screen. A developer-defined threshold triggers the notification.
 date: 2020-05-18
-updated: 2021-02-23
+updated: 2021-08-24
```
Comment (jpmedley) [inline src/site/content/en/blog/idle-detection/index.md:11]:
```suggestion
updated: 2021-08-25
```

File: src/site/content/en/blog/idle-detection/index.md (hunk 2 above)
Comment (jpmedley) [inline src/site/content/en/blog/idle-detection/index.md:49]:
```suggestion
| 5. **Launch**                            | **Chrome 94**            |
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Review by rowan-m — APPROVED** [review]
Looks as if the outstanding issues were suggestions from @jpmedley which appear to have been incorporated, so LGTM here.

**Pull request merged**